Description: use mktemp to create unique arbitrary file
 Create unique arbitrary file with mktemp rather than by hand.
Origin: debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2014-08-06

--- a/sbin/firehol.in
+++ b/sbin/firehol.in
@@ -2040,7 +2040,7 @@
 	local x=
 	for x in ${servers}
 	do
-		local tmp="${FIREHOL_DIR}/firehol.rpcinfo.$$.${RANDOM}"
+		local tmp="`${MKTEMP_CMD} ${FIREHOL_DIR}/firehol-rpcinfo-XXXXXXXXXX`"
 		
 		set_work_function "Getting RPC information from server '${x}'"
 		
@@ -2159,7 +2159,7 @@
 	local x=
 	for x in ${servers}
 	do
-		local tmp="${FIREHOL_DIR}/firehol.rpcinfo.$$.${RANDOM}"
+		local tmp="`${MKTEMP_CMD} ${FIREHOL_DIR}/firehol-rpcinfo-XXXXXXXXXX)`"
 		
 		set_work_function "Getting RPC information from server '${x}'"
 		
